src/release/release.yml

```diff
@@ -0,0 +1,29 @@
+# Orb Version 0.0.1
+
+version: 2.1
+description: Wrapper for Artsy's release management tools
+
+executors:
+  scripts:
+    docker:
+      - image: artsy/orb-scripts
```

Comment:
I don't see a `orb-scripts` Dockerfile in https://github.com/artsy/docker-images/ - would be good to commit it there

Reply:
Ahh, yeah. It's in the root of this repo. I've got it here because it copies over a bunch of bash scripts.